migrations/src/postDeploymentSetup.js

```diff
@@ -45,36 +42,21 @@ module.exports = async () => {
   if (NETWORK !== 'mainnet') {
     // Add new instance of StakeManager contract & Deployer address as Minter
 
-    const initialSupply = await schellingCoin.INITIAL_SUPPLY();
-
-    const mintableSupply = (BigNumber.from(10).pow(BigNumber.from(26))).mul(BigNumber.from(6));
-    const deployerBalance = BigNumber.from(await schellingCoin.balanceOf(signers[0].address));
-    const deployerSupply = BigNumber.from(initialSupply).sub(BigNumber.from(deployerBalance));
-
-    if (SCHELLING_COIN_ADDRESS !== '') {
-      // if previous instances of Schelling Coin is reused again and again,
-      // then initial balance will get depleted, thus intial tokens minting is needed,
-      // each time Schelling Coin instance is reused
-      await schellingCoin.addMinter(signers[0].address);
+    const initialSupply = (BigNumber.from(10).pow(BigNumber.from(26))).mul(BigNumber.from(6));
 
-      await schellingCoin.mint(signers[0].address, (deployerSupply));
-
-      // Remove previous instance of  Deployer address from Minter
-      await schellingCoin.removeMinter(signers[0].address);
-    }
-    await schellingCoin.transfer(stakeManagerAddress, mintableSupply);
+    await RAZOR.transfer(stakeManagerAddress, initialSupply);
```

Comment:
See As per this code
We can reuse old instance
The only issue is it won't have this much funds
as in the previous deployment we would have transferred 600M to Stakemanager.
Now again its not possible.
If we have to reuse the old instance we have to reuse the old instance of StakeManager as well, like upgradable contracts
Or we need fresh deployment every time
Or we can transfer it from old stakemanager to new one
Or we mint 
What do you think?

Reply:
@abhishekvispute I think we can tranfer it from old stakemanager to new one. I think we can create one new issue for this.

Reply:
Yes 
IMO
We should solve this in this PR only but
As without it, I will be blocked in the second deployment

If its taking time to solve this
I think we should revert to the fresh deployment of razor each time as of now
and we can work on solving this later 
CC: @SkandaBhat

Reply:
Sounds good.

Reply:
@abhishekvispute can you rescope this into a new issue?

Reply:
As per discussion we can 500-550k tokens.